bokehjs/src/coffee/renderer/tile/tile_renderer.coffee

```diff
@@ -24,6 +29,35 @@ class TileRendererView extends PlotWidget
     @_last_height = undefined
     @_last_width = undefined
 
+  _add_attribution: () =>
+    attribution = @mget('tile_source').get('attribution')
+
+    if _.isString(attribution) and attribution.length > 0
+      if @attributionEl?
+        @attributionEl.html(attribution)
+      else
+        border_width = 1
+
+        bottom_offset = @map_plot.get('min_border_bottom') + border_width
+        right_offset = @map_plot.get('min_border_right') + border_width
```

Comment:
I'm not sure if this is the right approach, I mean using `min_border_*`. @bryevdv, would you mind giving this a minute?